### ISSUE TITLE
LUDown cannot "parse toluis" for output files from command "ludown refresh" 

### ISSUE THREAD

**Issue by daden-ms** [microsoft/botbuilder-tools#1151]
## Tool
Name: [Msbot|Ludown|LUIS|QnAMaker|Chatdown|Dispatch|LuisGen]
Version: 1.3.1
OS:  Windows 10

## Describe the bug
I have a luis file which contains a phraselist "project" and a simple entity "project". After I converted luis json model file to .lu format, my attempt to convert the exact lu file that's generated from ludown refresh cannot be converted back to json format.  

## To Reproduce
1. create a luis model with one phraselist "project" and one simple entity 'project", download the json format
2. ludown refresh -i ./project_luss.json
3. cp ./project_luis.lu   ./project_luis_cp.lu
4. ludown parse  toluis --in  .\project_luis_cp.lu --verbose

## Expected behavior
a new project_luis_cp.json should be generated

## Screenshots

[ERROR]: 'Phrase List' entity: "project" is added as a labelled entity in utterance "find me all docs for project xxx". Phrase List cannot be added with explicit labelled values in utterances.
Stopping further processing. 

## Additional context

after I change phraselist name "project" to "projects", this error doesn't come up.

[bug]

**Comment by sgellock** [comment]
We need to consider this in the context of bf-cli

**Comment by aforank** [comment]
I am facing the same issue. However, if I pluralize the phraselist name as mentioned in the bug description then it works.

**Comment by litofish** [comment]
I'm still having this issue when a phraselist and simple entity have the same name. I've gotten it to work by pluralizing the phraselist, but should it also work without making this change?

**Comment by vishwacsena** [comment]
@litofish - can you verify that you are on ludown version 1.3.2? You can verify version via ```ludown -v``` and upgrade to the latest via ```npm i -g ludown```

The fix that went in to 1.3.2 should enable you to have the same name for a phrase list as well as a simple entity. 

As an example, this successfully parsed - 

```markdown
$ foo : phraseList 
    - test
    - one

$ foo : simple
```

```json
{
  "intents": [],
  "entities": [
    {
      "name": "foo",
      "roles": []
    }
  ],
  "composites": [],
  "closedLists": [],
  "regex_entities": [],
  "model_features": [
    {
      "name": "foo",
      "words": "test,one",
      "mode": false,
      "activated": true
    }
  ],
  "regex_features": [],
  "utterances": [],
  "patterns": [],
  "patternAnyEntities": [],
  "prebuiltEntities": [],
  "luis_schema_version": "3.0.0",
  "versionId": "0.1",
  "name": "testjson",
  "desc": "",
  "culture": "en-us"
}
```

**Comment by litofish** [comment]
I am on the latest version and still seem to be getting the error
![ludownerror](https://user-images.githubusercontent.com/39777667/62250794-c1435500-b3a2-11e9-9f6f-d2c1d466b031.PNG)

This is what I am doing:
```
## RequestItem
- i need more {Item=water}

$Item:simple

$Item:phraseList interchangeable
- water,coffee
```
